Replace deprecated function t3lib_div::array_merge_recursive_overrule

stweil opened the pull request:
That function was deprecated in TYPO3 6.2. Add a helper function which
calls the correct function depending on the TYPO3 version.

This approach was copied from the TYPO3 extension realurl.

Signed-off-by: Stefan Weil sw@weilnetz.de

sebastian-meyer commented:
I suggest adding the helper function in class `tx_dlf_helper` instead of `tx_dlf_document`. `tx_dlf_helper` already contains other functions which imitate old TYPO3 features (like `testInt()` or `intInRange()`).

stweil committed:
Replace deprecated function t3lib_div::array_merge_recursive_overrule

That function was deprecated in TYPO3 6.2. Add a helper function which
calls the correct function depending on the TYPO3 version.

This approach was copied from the TYPO3 extension realurl.

Signed-off-by: Stefan Weil <sw@weilnetz.de>

sebastian-meyer commented:
:+1:

stweil commented:
Now tx_dlf_helper is used as suggested. A runtime test is still to be done.

stweil commented:
The patch is now installed on our test system http://digi-test.bib.uni-mannheim.de/ and works, so feel free to pull it.
